docs/content/preview/explore/colocation.md

```diff
@@ -240,3 +240,133 @@ To set up xCluster for colocated tables, do the following:
+
+```sql
+\d table_name;
+```
+
+You should see output similar to the following:
+
+```output
+                 Table "public.t"
+ Column |  Type   | Collation | Nullable | Default
+--------+---------+-----------+----------+---------
+ col    | integer |           |          |
+Tablespace: "us_east_1a_zone_tablespace"
+Colocation: true
+```
+
+### List all tablegroups and associated tablespaces
+
+To list all tablegroups and their associated tablespaces, use the following query:
+
+```sql
+SELECT * FROM pg_yb_tablegroup;
+```
+
+You should see output similar to the following:
+
+```output
+  oid  |     grpname      | grpowner | grptablespace | grpacl | grpoptions
+-------+------------------+----------+---------------+--------+------------
+ 16387 | default          |       10 |             0 |        |
+ 16391 | colocation_16384 |       10 |         16384 |        |
+(2 rows)
+```
+
+The `oid` and `grpname` columns represent the tablegroup's OID and name, respectively. The `grptablespace` column shows the OID of the associated tablespace.
+
+### Geo-partitioned colocated tables with tablespaces
+
+YugabyteDB also supports [Row-level geo-partitioning](../multi-region-deployments/row-level-geo-partitioning/), which distributes each row across child tables based on the region specified by respective tablespaces. This capability enhances data access speed and helps meet compliance policies that require specific data locality.
+
+In a colocated database, you can now create colocated geo-partitioned tables to benefit from both colocation and geo-partitioning simultaneously as follows:
+
+```sql
+CREATE TABLE <partitioned_table_name> PARTITION BY LIST(region);
+
+CREATE TABLE <partition_table1> PARTITION OF <partitioned_table_name> FOR VALUES IN (<region1>) TABLESPACE <tablespace1>;
+
+CREATE TABLE <partition_table2> PARTITION OF <partitioned_table_name> FOR VALUES IN (<region2>) TABLESPACE <tablespace2>;
+
+CREATE TABLE <partition_table3> PARTITION OF <partitioned_table_name> FOR VALUES IN (<region3>) TABLESPACE <tablespace3>;
+```
+
+### Alter tablespaces for colocated relations
+
+Colocated relations (the strategy of storing related data together to optimize performance and query efficiency) cannot be moved independently from one tablespace to another; instead, you can either move all colocated relations or none.
+
+To move all relations from one tablespace to another, use the following syntax:
+
+```sql
+ALTER TABLE ALL IN TABLESPACE tablespace_1 SET TABLESPACE tablespace_2 CASCADE;
+```
+
+This command moves all relations in `tablespace_1` to `tablespace_2`, including any non-colocated relations.
+
+To move only colocated relations, you can specify a table to colocate with using the following syntax:
+
+```sql
+ALTER TABLE ALL IN TABLESPACE tablespace_1 COLOCATED WITH table1 SET TABLESPACE tablespace_2 CASCADE;
+```
+
+This command will move only the tables present in `tablespace_1` that are colocated with `table1`.
```

Comment:
This command will move only the ~tables~ relations present in `tablespace_1` that are colocated with `table1`